emails/management/commands/send_welcome_emails.py

```diff
@@ -60,11 +61,16 @@ def send_welcome_email(profile: Profile, **kwargs):
                 },
             },
         )
+        logger.info(f"Sent welcome email to user ID: {profile.user.id}")
+        profile.sent_welcome_email = True
+        profile.save()
     # Don't send welcome emails to users with no social account.
     # E.g., users created thru admin tools.
     # TODO?: move this check deeper into get_welcome_email ?
     except SocialAccount.DoesNotExist:
-        pass
-
-    profile.sent_welcome_email = True
-    profile.save()
+        profile.sent_welcome_email = True
+        profile.save()
+    except ClientError:
+        logger.error(
```

Comment:
This user will raise the error again next time. Is the intention to get data before figuring out automated actions?

Reply:
Yes, I wasn't sure how many of these we might have so I didn't want to just ignore them all completely. When we see how often this happens and what kind of addresses cause it, we can decide if we just skip these welcome emails.